gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.3-all.zip
```

Comment:
You might need to run `./gradlew wrapper --gradle-version=8.3` once or twice more since I believe the jar and gradle wrapper executable have changed.

Reply:
Done!